docs/quickstarts/ios.mdx

```diff
@@ -0,0 +1,296 @@
+---
+description: Add authentication and user management to your iOS app with Clerk.
+---
+
+# iOS Quickstart (Beta)
+
+<Callout type="warning">
+  The Clerk iOS SDK is currently in beta. It is **not yet recommended for production use**.
+</Callout>
+
+<TutorialHero
+  framework="iOS"
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk application",
+      link: "https://clerk.com/docs/quickstarts/setup-clerk",
+      icon: "clerk",
+    }
+  ]}
+>
+  - Create a new Xcode project
+  - Install the Clerk iOS SDK
+  - Configure `Clerk` with your publishable key
+  - Use Clerk to enable users to sign in and out of your application
+</TutorialHero>
+
+<Steps>
+  ### Create an iOS Project
+
+  To get started using Clerk with iOS, create a new project in Xcode. Select SwiftUI as your interface and Swift as your language.
+  See the [Xcode documentation](https://developer.apple.com/documentation/xcode/creating-an-xcode-project-for-an-app) for more information.
+
+  ### Install the Clerk iOS SDK
+
+  Follow the Swift Package Manager [instructions](https://developer.apple.com/documentation/xcode/adding-package-dependencies-to-your-app) to install Clerk as a dependency.
+  When prompted for the package URL, enter [https://github.com/clerk/clerk-ios](https://github.com/clerk/clerk-ios). Be sure to add the package to your target.
+
+  ### Load Clerk
+
+  To use Clerk in your app, you must first configure and load `Clerk`.
+
+  - Inside your new project in Xcode, open your `@main` app file.
+  - Import `ClerkSDK`.
+  - Add a new variable `clerk` marked with the `@ObservedObject` property wrapper.
+  - Replace `ContentView()` with the below `ZStack`.
+  - Attach a `.task` modifier to the new `ZStack`. In the body of the task, configure `clerk` with your publishable key, and then load `clerk`. You can get your publishable key from the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page of the Clerk Dashboard.
+
+  <InjectKeys>
+    ```swift {{ filename: 'ClerkQuickstartApp.swift', mark: [2, 6, [10, 20]] }}
+    import SwiftUI
+    import ClerkSDK
+
+    @main
+    struct ClerkQuickstartApp: App {
+        @ObservedObject private var clerk = Clerk.shared
+
+        var body: some Scene {
+            WindowGroup {
+                ZStack {
+                    if clerk.loadingState == .notLoaded {
+                        ProgressView()
+                    } else {
+                        ContentView()
+                    }
+                }
+                .task {
+                    clerk.configure(publishableKey: "pk_test_YW11c2luZy1iYXJuYWNsZS0yNi5jbGVyay5hY2NvdW50cy5kZXYk")
+                    try? await clerk.load()
+                }
+            }
+        }
+    }
+    ```
+  </InjectKeys>
```

Comment:
`<InjectKeys>` is not required, and the [`{{pub_key}}` shortcode](https://github.com/clerk/clerk-docs/blob/main/CONTRIBUTING.md#code-block-shortcodes) can be used to inject the user's public key:

```suggestion
  ```swift {{ filename: 'ClerkQuickstartApp.swift', mark: [2, 6, [10, 20]] }}
  import SwiftUI
  import ClerkSDK

  @main
  struct ClerkQuickstartApp: App {
      @ObservedObject private var clerk = Clerk.shared

      var body: some Scene {
          WindowGroup {
              ZStack {
                  if clerk.loadingState == .notLoaded {
                      ProgressView()
                  } else {
                      ContentView()
                  }
              }
              .task {
                  clerk.configure(publishableKey: "{{pub_key}}")
                  try? await clerk.load()
              }
          }
      }
  }
  ```
```

Reply:
oh thanks good catch! I had {{ pubkey }} in there but I must've copy pasted over it at some point.